predicate/predicate_paging.go

```diff
@@ -0,0 +1,260 @@
+package predicate
+
+import (
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client/internal/hzerrors"
+	"github.com/hazelcast/hazelcast-go-client/internal/proto"
+	iserialization "github.com/hazelcast/hazelcast-go-client/internal/serialization"
+	"github.com/hazelcast/hazelcast-go-client/internal/util/nilutil"
+	"github.com/hazelcast/hazelcast-go-client/serialization"
+	"github.com/hazelcast/hazelcast-go-client/types"
+)
+
+type predPaging struct {
+	comparator        serialization.IdentifiedDataSerializable
+	internalPredicate Predicate
+	iterationType     IterationType
+	anchorList        []types.Entry
```

Comment:
How about using shorter names for fields, such as: `cmp, pred, iterType, anchors`?

Reply:
Addressed in d226845